dev/ProgressRing/ProgressRing.xaml

```diff
@@ -15,35 +14,26 @@
         <Setter Property="MinWidth" Value="16" />
         <Setter Property="IsTabStop" Value="False" />
         <Setter Property="Width" Value="32" />
+        <Setter Property="Maximum" Value="100" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:ProgressRing">
                     <Grid x:Name="LayoutRoot" Background="Transparent">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Active">
-                                    <Storyboard>
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="LayoutRoot"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="1"
-                                            Duration="0:0:0.15" />
-                                    </Storyboard>
-                                </VisualState>
                                 <VisualState x:Name="Inactive">
-                                    <Storyboard>
-                                        <DoubleAnimation
-                                            Storyboard.TargetName="LayoutRoot"
-                                            Storyboard.TargetProperty="Opacity"
-                                            To="0"
-                                            Duration="0" />
-                                    </Storyboard>
+                                    <VisualState.Setters>
+                                        <Setter Target="LayoutRoot.Opacity" Value="0" />
```

Comment:
Can you explain this? I don't get why the whole control is hidden here.

Reply:
The entire control is designed to be hidden when `IsActive` is false. `LayoutRoot` is the setter target because 1) there used to be 2 AVPs, 2) for backwards compatibility with WUXC.